contrib/mod_sftp/mac.c

```diff
@@ -877,9 +877,9 @@ const char *sftp_mac_get_write_algo(void) {
 
   /* It is possible for there to be no MAC, as for some ciphers such as
    * AES-GCM.  Rather than returning NULL here, we indicate this by returning
-   * the empty string (see Issue #1411).
+   * a string (see Issue #1411).
    */
-  return "";
+  return "implicit";
```

Comment:
These just make this `mac.c` implementation match what is logged during KEX, for consistency.